### PR TITLE
Spectator camera clean up

### PR DESCRIPTION
**Pull request by manapa01**
Code review and cleaning up comments to make them more compact and easier to understand. Small clarifications of variables and functionality.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1141/HighFidelity-Beta-PR10834-994117b524eb0ad16a55a95393238bafbca185e6.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1141/HighFidelity-Beta-PR10834-994117b524eb0ad16a55a95393238bafbca185e6.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1144/HighFidelity-Beta-PR10834-1392297da9b2ee5ff22ad90de50bec6c9dcb461a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1144/HighFidelity-Beta-PR10834-1392297da9b2ee5ff22ad90de50bec6c9dcb461a.dmg

File: scripts/system/spectatorCamera.js [hunk 1]
```diff
@@ -1,7 +1,4 @@
 "use strict";
-/*jslint vars:true, plusplus:true, forin:true*/
-/*global Tablet, Script,  */
-/* eslint indent: ["error", 4, { "outerIIFEBody": 0 }] */
```
Comment (howard-stearns) [inline scripts/system/spectatorCamera.js]: why remove this? we generally want people to use linters.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1160/HighFidelity-Beta-PR10834-9bcaa6d2566bde83fb2a8febeaf0724add76600c.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/1162/HighFidelity-Beta-PR10834-cb52122726b135a87b3392d6b6aba551afdb0e88.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/1162/HighFidelity-Beta-PR10834-cb52122726b135a87b3392d6b6aba551afdb0e88.exe

**Pull request merged**